test/e2e_node/conformance/build/Makefile

```diff
@@ -0,0 +1,94 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Build the node-test image.
+#
+# Usage:
+#   [ARCH=amd64] [REGISTRY="gcr.io/google_containers"] [BIN_DIR="../../../../_output/bin"] make (build|push) VERSION={some_version_number e.g. v0.1}
+
+# TODO(random-liu): Add this into release progress.
+REGISTRY?=gcr.io/google_containers
+ARCH?=amd64
+# BIN_DIR is the directory to find binaries, overwrite with ../../../../_output/bin
+# for local development.
+BIN_DIR?=../../../../_output/dockerized/bin/linux/${ARCH}
+TEMP_DIR:=$(shell mktemp -d)
+
+UNAME_S:=$(shell uname -s)
+ifeq ($(UNAME_S),Darwin)
+	SED_CMD?=sed -i ""
+endif
+ifeq ($(UNAME_S),Linux)
+	SED_CMD?=sed -i
+endif
+
+ifeq ($(ARCH),amd64)
+       BASEIMAGE?=debian:jessie
```

Comment:
`busybox:glibc` does exist for all arches (alpine does not)
busybox:glibc does not include libpthread though, so if the test requires that lib, we have to go with debian or ubuntu until all binaries are statically linked

user.Current() was one of the problems I encountered in #26028